src/pages/mine/index.tsx

```diff
@@ -91,39 +89,8 @@ const ContentDev = () => {
     );
 };
 
-function CardList({ tasks }: { tasks: task[] }) {
-    return (
-        <>
-            {tasks.map((item) => (
-                <Card
-                    content={item}
-                    key={item.id}
-                    shouldEdit={false}
-                    onContentChange={undefined}
-                />
-            ))}
-        </>
-    );
-}
```

Comment:
Are we sure, that this is not being used any where?

Reply:
Yes @prakashchoudhary07, I have double checked. It is not being used anywhere.